Package@swift-6.0.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.11
+// swift-tools-version: 6.0
```

Comment:
We need to keep a -5.11 manifest around for folks who don't have a bleeding-edge toolchain installed.

Reply:
Thinking about that further, if we _don't_ keep a 5.11 manifest around, I think the effect is that SPM 5.11 will use the 5.10 manifest, right? So the only real consequence is that Git information will be unavailable, which is fine.